diff --git a/node_modules/usb/libusb/libusb/os/windows_usb.c b/node_modules/usb/libusb/libusb/os/windows_usb.c
index 31389af..d0af000 100644
--- a/node_modules/usb/libusb/libusb/os/windows_usb.c
+++ b/node_modules/usb/libusb/libusb/os/windows_usb.c
@@ -567,7 +567,8 @@ static SP_DEVICE_INTERFACE_DETAIL_DATA_A *get_interface_details_filter(struct li
 static char *parse_device_interface_path(const char *interface_path)
 {
 	char *device_id, *guid_start;
-	unsigned int i, len;
+	unsigned int i;
+	int len;
 
 	len = safe_strlen(interface_path);
 	if (len < 4) {
